src/CommunityToolkit.Maui/Layouts/StateContainer/StateContainerController.shared.cs

```diff
@@ -8,7 +8,7 @@ sealed class StateContainerController
 	readonly WeakReference<Layout> layoutWeakReference;
 
 	string? previousState;
-	List<View> originalContent = Enumerable.Empty<View>().ToList();
+	IList<IView> originalContent = Enumerable.Empty<IView>().ToList();
```

Comment:
this's a private field and isn't injected anywhere. So, I would say it's fine to use the concrete type here.

Reply:
Hey Pedro! Are you advocating to continue using `View` instead of replacing it with `IView`? 

I made the change to `IView` so that we could remove the casting here: https://github.com/CommunityToolkit/Maui/pull/1262/files#diff-c16a4976748b4af82ec28108ed444db5aa96141fef8d3a90e3b9769e2ea872e9L55-R55

Reply:
No, using `List<IView>`, instead of `IList<IView>`

Reply:
```suggestion
	List<IView> originalContent = Enumerable.Empty<IView>().ToList();
```